src/main/java/com/learning/libraryManagement/controller/UsersController.java

```diff
@@ -0,0 +1,26 @@
+package com.learning.libraryManagement.controller;
+
+import com.learning.libraryManagement.model.Users;
+import com.learning.libraryManagement.service.UsersService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class UsersController {
+    @Autowired
+    UsersService usersService;
+
+    @PostMapping("/createUsers")
+
+    public List<Users> createMultipleUserList(@RequestBody List<Users> userList) {
```

Comment:
This method name is not yet changed to createUsers

Reply:
Done